kratos/python/add_utilities_to_python.cpp

```diff
@@ -839,6 +840,9 @@ void AddUtilitiesToPython(pybind11::module &m)
     m.def("InvertNormal",&MortarUtilities::InvertNormal<PointerVectorSet<Element, IndexedObject>>);
     m.def("InvertNormal",&MortarUtilities::InvertNormal<PointerVectorSet<Condition, IndexedObject>>);
 
+    // Delaunator utilities
+    m.def("CreateTriangleMeshFromNodes",&DelaunatorUtilities::CreateTriangleMeshFromNodes);
```

Comment:
I would say this should be exposed as a submodule, like [here](https://github.com/KratosMultiphysics/Kratos/blob/master/kratos/python/add_utilities_to_python.cpp#L914-L989)

Reply:
ping